_posts/2019-03-19-Kickstarter-Trajectory-2019-Light.md

```diff
@@ -0,0 +1,296 @@
+
+## Cloud Providers & Platforms
+<img class="image left medium" alt="AWS Logo" src="/img/2019-Kickstarter-Trajectory-2019-light/aws-logo.png">
+To get a bigger picture of all the cloud providers and platforms that are out there conquering the IT world, 
+we had a dedicated session about this topic given by [Bas Moorkens](/author/bas-moorkens) and [Dieter Hubau](/author/dieter-hubau).
+Bas was focusing on Amazon Web Services and all its features that it has to offer.
+
+We quickly learned that AWS was very advanced and had lots of components to facilitate the life of a developer.
+It was a very interesting session and made me realise that AWS was a big part of the development scene.
+We are eager to use it and learn more of what is has to offer. 
+
+<img class="image right medium" alt="OpenShift Logo" src="/img/2019-Kickstarter-Trajectory-2019-light/openshift-logo.png">
+
+As for cloud platforms, we got a very detailed explanation of how OpenShift (made by RedHat) works and what its features and options are.
+We also got a high-level explanation as to how an application in the cloud works and what the best practices are to achieve deploying your application in the cloud.
+
+Overall, it was a very interesting session for cloud-enthusiasts and we definitely want to learn more about it!
+
+## TypeScript
+After the session HTLML5, CSS3, JavaScript, [Dimitri De Kerf](/author/dimitri-de-kerf) learned us the benefits of TypeScript. 
+He told us some benefits of using TypeScript instead of JavaScript. This means that TypeScript is a wrapper around JavaScript, 
+which means it has more methods to use to develop a application which you otherwise would not be possible.
+
+Like declaring declaring a colon and a type after the variable.
+
+[Dimitri De Kerf](/author/dimitri-de-kerf) showed how to configure our project to use TypeScript and to use these features.
+He explained us that it is important to know how to use TypeScript because it is used in popular frameworks like Angular and React.
+
+
+## Angular
+
+<img class="image right" width="300" alt="Angular logo" src="/img/2019-Kickstarter-Trajectory-2019-light/angular.png">
+
+[Ryan De Gruyter](/autor/Ryan-De-Gruyter) was our teacher for today.
+He quickly introduced us to Angular, a platform that is designed to easily create browser applications built by Google. 
+The first version of Angular was AngularJS, it was very popular and used by many companies.
+Google decided to update Angular and created Angular 2 which was not welcomed by the industry at that time because it removed all the AngularJS concepts. 
+
+It took some time for the industry to adapt and see the positive stuff of Angular2: Open Source community, improved dependency injection, better performance, enc..
+Angular 2 is not the holy grail of frameworks. It still has some downsides like lots of ceremony and boilerplate thanks to the use of Angular CLI.
```

Comment:
Lots of ceremonies?

Reply:
Not sure. Don't think so though, according to this post: 
https://wildermuth.com/2016/07/31/Is-Angular2-Too-Full-of-Ceremony

Reply:
I see. Hadn't heard about that saying. TIL... :-)